Source/Renderer/Context.js

```diff
@@ -276,6 +276,10 @@ define([
         this._fragDepth = !!getExtension(gl, ['EXT_frag_depth']);
         this._debugShaders = getExtension(gl, ['WEBGL_debug_shaders']);
 
+        if (this._webgl2) {
+            this._colorBufferFloat = !!getExtension(gl, ['EXT_color_buffer_float']);
```

Comment:
Make this:
`this._colorBufferFloat = this._webgl2 && !!getExtension(gl, ['EXT_color_buffer_float']);`